VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+# Validators FAQ
+
+## General concepts
+
+### What is a validator?
+
+Validators are the pillars of the network. Their role is to run a full node and participate in consensus by broadcasting votes which contain cryptographic signatures signed by their private key. Validators relay themselves to commit new blocks in the blockchain and receive rewards in exchange for their work. They must also participate in governance by voting on proposals.
+
+### What is a full node?
+
+A full node is a program that fully validates transactions and blocks of a blockchain. It is distinct from a light node that only processes block headers and a small subset of transactions. Running a full node requires more resources than a light-node but is necessary in order to be a validator. In practice, running a full node only implies running a non-compromised and up-to-date version of the software with low network latency and without down time. 
```

Comment:
Let's be consistent with the spelling of `light node`. I think it should be `light-node` and `full-node`.

Reply:
A light-node does not process block headers. Well it might, but it does not have to process every single block header. 
``
A light-node is able to verify any data it receives from any full-node (whether malicious or not) and hence is able to present the end-user with a verified view of the current state of the network or an error (in the case of a malicious node).
```

Reply:
How can you synch up from latest checkpoint if you don't process block headers ?